src/lib.rs

```diff
@@ -165,6 +165,8 @@ pub struct FormatReport {
     non_formatted_ranges: Vec<(usize, usize)>,
 }
 
+unsafe impl Send for FormatReport {}
```

Comment:
out of curiosity, why is the unsafe needed here ?

Reply:
Send cannot be implemented without unsafe

Reply:
`FormatReport` is part of public API, so I prefer not to change it. Also, `FormatReport` should not be marked as `Send` as it uses `Rc`. You need to replace it with `Arc`.